app/models/sections/transportUnit/TransportUnitsAddToListModel.scala

```diff
@@ -46,9 +46,9 @@ object TransportUnitsAddToListModel extends Enumerable.Implicits {
 
     Seq(
       radioItem(Yes, 0),
-      radioItem(MoreToCome, 1),
+      radioItem(NoMoreToCome, 1),
       orDivider,
-      radioItem(NoMoreToCome, 2)
+      radioItem(MoreToCome, 2)
```

Comment:
has the order changed?

Reply:
Screenshot from prototype:

![image](https://github.com/hmrc/emcs-tfe-create-movement-frontend/assets/31282758/9a9c31c4-e045-4f88-89f6-6656e5f95a88)

More to come should've been the last one, the underlying numbers were just the wrong way around even though they looked the same 🙂

Reply:
It was the last one in the demo, not sure why / when it changed

Reply:
Ahhh